RevenueCatUI/Templates/Components/TemplateComponentsView.swift

```diff
@@ -43,9 +43,10 @@ public struct TemplateComponentsView: View {
 
     public var body: some View {
         VStack(spacing: 0) {
-                ComponentsView(
-                    componentViewModels: self.componentViewModels
-                )
+            ComponentsView(
+                componentViewModels: self.componentViewModels
+            )
+            Spacer()
```

Comment:
I assume this is here to expand and make things flow down from top to bottom rather than centre-out, but as the paywall fills up, this could add extra unwanted spacing at the bottom.

Does replacing it with a `.frame(maxHeight: .infinity, alignment: .top)` on `ComponentView(` have the desired effect, without adding a possibly problematic spacer at the bottom?